test/controllers/PartsControllerSpec.scala

```diff
@@ -96,4 +96,13 @@ class PartsControllerSpec extends FlatSpec with Matchers with MockitoSugar with
     contentAsString(result) shouldNot include("slow")
   }
 
+  it should "show a filtered list of octoparts when POST is used" in {
+    val result = controller.listPost.apply(FakeRequest().withJsonBody(Json.obj("ids" -> Json.arr("void"))))
+
+    status(result) should be(200)
+    contentAsString(result) should include("void")
+    contentAsString(result) shouldNot include("error")
+    contentAsString(result) shouldNot include("slow")
+  }
```

Comment:
Yes, if you POST without a filter, it behaves the same way the GET would; no filter is applied.